exercises/en/test_03_09b.py

```diff
@@ -0,0 +1,22 @@
+def test():
+    # Here we can either check objects created in the solution code, or the
+    # string value of the solution, available as __solution__. A helper for
+    # printing formatted messages is available as __msg__. See the testTemplate
+    # in the meta.json for details.
+    # If an assertion fails, the message will be displayed
+
+    # Since we haven't started assigning charts to variable names yet,
+    # this might be the better way to test for the first exercise.
+    # Maybe even for later exercises.
+    assert 'alt.Chart' in __solution__, "Make sure you are calling the 'Chart' function?"
+    assert 'mark_area' in __solution__, "Make sure you are creating a density plot using 'mark_area'."
+    assert "groupby=" in __solution__, "Make sure you are grouping by 'island' and 'species'."
+    assert "steps=100" in __solution__, "Make sure you are using  steps=100 when computing the KDE."
```

Comment:
Can you check both of these in the encodings?

Reply:
Again, encodings are not present for faceted plots.